### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by yordv**
Fix broken links for PSR-1 and PSR-2

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ First of all, **thank you** for contributing, **you are awesome**!
 Here are a few rules to follow in order to ease code reviews, and discussions before
 maintainers accept and merge your work.
 
-You MUST follow the [PSR-1](http://www.php-fig.org/psr/1/) and
-[PSR-2](http://www.php-fig.org/psr/2/). If you don't know about any of them, you
+You MUST follow the [PSR-1](https://www.php-fig.org/psr/psr-1/) and
```
Comment (garak) [inline CONTRIBUTING.md]: What about omitting protocol?

Comment (yordv) [reply CONTRIBUTING.md]: Its redirect to https if you hit http

Comment (garak) [reply CONTRIBUTING.md]: Exactly. So we can link to `//www.php-fig.org/psr/psr-1/` instead of `https://www.php-fig.org/psr/psr-1/`

**Comment by garak** [comment]
Thank you!

**Pull request merged**